didcomm/errors.py

```diff
@@ -0,0 +1,45 @@
+from enum import Enum
+
+
+# TODO: finalize exceptions during development
+
+class DIDCommError(Exception):
```

Comment:
Common error usage here looks unsafe for me.

Reply:
Having a common class for all custom errors is a standard practice for custom exceptions in Python. It can be useful in some cases (such as a demo) just to catch a base DIDCommError instead of catching every individual exception.